README.md

```diff
@@ -24,7 +24,6 @@ Installation
 
 5. Create the database schema:
 
-        $ cd handoverservice
         $ python manage.py migrate
 
 6. Start the app:
```

Comment:
You may want to consider adding instructions for docker if that is the preferred method of running this in production.